groups.sql

```diff
@@ -0,0 +1,331 @@
+BEGIN TRANSACTION;
+CREATE TABLE IF NOT EXISTS "groups" (

```

Comment:
Can you explain what `IF NOT EXISTS` does?

Reply:
checks if table already exists or not to activate this command i guess